src/components/FilterHeader.js

```diff
@@ -27,6 +25,10 @@ type State = {
 };
 
 class FilterHeader extends React.PureComponent<Props, State> {
+  static defaultProps = {
+    onFilterCategoriesPress: () => {}
```

Comment:
Do we really need a default prop for this? In the `Props` it looks like it is required.